src/Jaeger.Core/Tracer.cs

```diff
@@ -288,7 +299,7 @@ public Builder WithTag(string key, string value)
                 return this;
             }
 
-            private Builder WithTags(IDictionary<string, object> tags)
+            public Builder WithTags(IEnumerable<KeyValuePair<string, string>> tags)
```

Comment:
Should we keep a variante with `IEnumerable<KeyValuePair<string, object>>` or are we sure we will only have string pairs? I assume this was changed for `Configuration.GetTracerBuilder`.

Reply:
I changed this because Java also only has a `String, String` method.

Reply:
Still looks strange since the receiving dictionary is <string, object>. Is java using <string, string>? (Can’t look right now, just boarding my plane to Graz).

Reply:
yes, they are using <string, string>.